test/helper/fake_cloud_controller_api.go

```diff
@@ -2823,6 +2834,118 @@ func (f *FakeCloudControllerAPI) writeResponse(rw http.ResponseWriter, r *http.R
 					]
 				}`)))
 		}
+	case "/v2/quota_definitions":
+		rw.Write([]byte(fmt.Sprintf(`{
```

Comment:
Something i wanted to do here (for clean up) was to put all those JSON in resource files and load them here instead.

Reply:
Agreed, but let's do that in a separate PR.